builder/xenserver/common/common_config.go

```diff
@@ -77,7 +77,7 @@ func (c *CommonConfig) Prepare(ctx *interpolate.Context, pc *common.PackerConfig
 	}
 
 	if c.ToolsIsoName == "" {
-		c.ToolsIsoName = "xs-tools.iso"
+		c.ToolsIsoName = ""
 	}
```

Comment:
Since we don't want to apply a default value, removing this entirely will have the desired effect.

Reply:
And done :D